docs/API_reference.md

```diff
@@ -562,6 +564,36 @@ There are currently two supported output formats: 'json' and 'dataframe'.
 The 'json' format simply returns the raw response, without page information and with multiple pages combined into a single data structure.  
 The 'dataframe' format transforms that data structure into a pandas DataFrame, which contains subparts in json, that can be split further by the user application.
 
+#### Plenty BI related data <a name='get-bi-section'></a>
+
+##### plenty_api_get_contacts <a name='get-bi'></a>
+
+Get list of raw data files and (optionally) download raw data.
+
+[*Optional parameter*]:
+**dataName**:  restricts the search results to raw data files e.g. orders,orderItems.
+Multible raw data files can be separated by comma.
+
+**processStatus**:  Current process status, the status is only changed by internal processing. If this filter is used, the filter dataNames may only contain one value.
+
+**createdAtTimestamp**: UNIX-Timestamp from when daily generated raw data are to be filtered. This filter cannot be combined with the following filters: dataNames, processStatus
+
+**itemsPerPage**: The number of raw data files to be returned. In this implemetation the maximum of 100 items is set to default (instead of Plenty's 20)
```

Comment:
s/In this implemetation the maximum of 100 items is set to default (instead of Plenty's 20)/This library uses a default value of 100 instead of 20, which is the default for the REST API./